enterprise/frontend/src/metabase-enterprise/audit_app/pages/AuditDashboardDetail.jsx

```diff
@@ -23,7 +23,7 @@ const AuditDashboardDetail = ({ params, ...props }: Props) => {
     <AuditContent
       {...props}
       title={<EntityName entityType="dashboards" entityId={dashboardId} />}
-      subtitle={<OpenInMetabase to={Urls.dashboard(dashboardId)} />}
+      subtitle={<OpenInMetabase to={Urls.dashboard({ id: dashboardId })} />}
```

Comment:
Missed that 😢 Thanks for fixing that. Is there a way we can pass the whole dashboard object or its name to `Urls.dashboard`? In that way, it'd be possible to append a slug to its URL

Reply:
Please LMK if it's not an easy thing to do, I can dig into that later

Reply:
Added slugs, thanks for suggesting it!

Reply:
Thank you!